modules/serverless-build-func-kn.adoc

```diff
@@ -8,29 +8,50 @@
 
 Before you can run a function, you must build the function project by using the `kn func build` command. This command creates an OCI container image that can be run locally on your computer or on an {product-title} cluster. The build command uses the function project name and the image registry name to construct a fully qualified image name for your function.
```

Comment:
It might be worth mentioning that `func build` is something that a user might use for debugging, or for more advanced usage, but in general a user should use `func create` and `func deploy` for most tasks, and `fund deploy` actually performs the build.